src/server/utils/api.ts

```diff
@@ -82,6 +82,24 @@ API.get('/portfolio/:userName', async (_req, res) => {
   }
 });
 
+//consulta de la Api
+API.get('/portfolios', async (_req, res) => {
+  try {
+    const portfolioRef = await db.collection('Portfolio').get();
+    const portfolios: Portfolio[] = [];
+
+    portfolioRef.forEach(doc => {

```

Comment:
we don't do a forEach for pushing elements into another array.

better to use `map`

suggestion: const portfolios = portfolioRef.map(doc => (doc.data()) as Portfolio)